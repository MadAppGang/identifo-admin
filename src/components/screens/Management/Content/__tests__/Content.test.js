import ManagementScreenContent from '..';

describe('<ManagementScreenContent />', () => {
  test('renders as expected', () => {
    expect(shallow(<ManagementScreenContent />)).toMatchSnapshot();
  });

  test('renders children when passed in', () => {
    const component = shallow(
      <ManagementScreenContent>
        <div>Hello, world!</div>
      </ManagementScreenContent>,
    );

    expect(component.contains(<div>Hello, world!</div>)).toBe(true);
  });
});
