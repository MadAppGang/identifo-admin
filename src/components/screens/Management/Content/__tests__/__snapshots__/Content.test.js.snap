// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ManagementScreenContent /> renders as expected 1`] = `
<div
  className="iap-management-content"
/>
`;
